### PR TITLE
[260327] resolve todo comments

### PR DESCRIPTION
**Pull request by aSyoriKano**
https://acsmine.tok.access-company.com/redmine/issues/260327

### CODE REVIEW

**Review by aYukiSekiguchi — APPROVED** [review]
LGTM

File: lib/test/config.ex [hunk 1]
```diff
@@ -29,13 +26,12 @@ defmodule Antikythera.Test.Config do
     if blackbox_test?() do
       ExUnit.configure(
         exclude: [:test],
-        include: [:blackbox, :blackbox_only],
-        module_load_timeout: @long_module_load_timeout
+        include: [:blackbox, :blackbox_only]
       )
 
       IO.puts("Target base URL: #{base_url()}")
```
Comment (Double-oxygeN) [inline lib/test/config.ex:32]: Eliminating these line breaks would help readability, but I will respect your decision.
This change does not affect the result of `mix format`.

```diff
diff --git a/lib/test/config.ex b/lib/test/config.ex
index b18f3d8..c5e70bc 100644
--- a/lib/test/config.ex
+++ b/lib/test/config.ex
@@ -24,11 +24,7 @@ defmodule Antikythera.Test.Config do
     ExUnit.start()
 
     if blackbox_test?() do
-      ExUnit.configure(
-        exclude: [:test],
-        include: [:blackbox, :blackbox_only]
-      )
-
+      ExUnit.configure(exclude: [:test], include: [:blackbox, :blackbox_only])
       IO.puts("Target base URL: #{base_url()}")
     else
       ExUnit.configure(exclude: [:blackbox_only])
```

**Review by Double-oxygeN — APPROVED** [review]
LGTM

File: lib/test/config.ex (hunk 1 above)
Comment (aSyoriKano) [reply lib/test/config.ex]: Thanks for suggestion, I prefer your code.
Fixed in cac2c80

**Pull request merged**